Tests/test_image_access.py

```diff
@@ -1,15 +1,21 @@
 from helper import unittest, PillowTestCase, hopper, on_appveyor
 
-try:
-    from PIL import PyAccess
-except ImportError:
-    # Skip in setUp()
-    pass
-
 from PIL import Image
 import sys
 import os
 
+# CFFI imports pycparser which doesn't support PYTHONOPTIMIZE=2
+# https://github.com/eliben/pycparser/pull/198#issuecomment-317001670
+if os.environ.get("PYTHONOPTIMIZE") == "2":
+    cffi = None
+else:
+    try:
+        from PIL import PyAccess
+        import cffi
+    except ImportError:
+        cffi = None
+        pass
```

Comment:
Unless you feel otherwise, `pass` is unnecessary here

Reply:
Thanks, removed.